app/models/appointment.rb

```diff
@@ -1,4 +1,14 @@
 class Appointment < ActiveRecord::Base
+  enum status: %i(
+    pending
+    completed
+    no_show
+    ineligible_age
+    ineligible_pension_type
+    cancelled_by_customer
+    cancelled_by_pension_wise
+  )
```

Comment:
tl;dr I'm OK with this as-is.

We won't be integrating directly with the database, so storing the names of the statuses in a denormalised way makes no sense here. Using a lookup table is adding another unnecessary join, and we don't need the lookups to be editable.

With regards ordering in enums - if you change the ordering of an enum then you shouldn't be trusted to write code. Enums are inherently ordered structures. 😄